examples/3-twitter-clone/src/app/models/RootStore.ts

```diff
@@ -22,13 +26,13 @@ export const RootStore = RootStoreBase.props({
 })
   .views(self => ({
```

Comment:
for some reason all the `self`'s on rootStore are giving me a `Parameter 'self' implicitly has an 'any' type, but a better type may be inferred from usage.`